docs/ontology/ptf/v2/CategoryClaim.md

```diff
@@ -10,7 +10,6 @@ A generic Proof Point type that can be used to describe many types of claim, as
 [{
   id: "https://provenance.org/users/example",
   category: "example category",
-  statement: "example statement",
```

Comment:
If statement is being removed, should initiatives also be removed?

Has this been discussed with Product?